### DISCUSSION TITLE
FormTab下当tab传入自定义组件时，无法隐藏

### DISCUSSION THREAD

**Discussion by PlutoCA** [alibaba/formily#2465] [Q&A 2.x] [answered]
FormTab 下面当tab传入自定义组件时 无法操作隐藏
期望：正常控制显示隐藏

https://codesandbox.io/s/blazing-bush-2ycui?file=/App.tsx

![image](https://user-images.githubusercontent.com/31099607/142584622-b88c78d2-0c1c-4f3f-bb13-fe2b6ce3cfe3.png)
![image](https://user-images.githubusercontent.com/31099607/142584778-a1d1164a-aa9c-4843-9bfe-caf9e112e53c.png)

自定义方式传入组件就不行，操作隐藏会失败。
不确定是不是我写法的问题，还是不支持这种方式。
（如果不支持这种方式，有其它的代替方案么）
请教一下大佬，帮忙看看。感谢~

**Comment by janryWang** [comment, marked as answer]
没看到你例子里有FormTab呀

**Reply by PlutoCA** [reply]
![image](https://user-images.githubusercontent.com/31099607/142591889-dbb97d13-4c74-4bfa-91a1-c3c0b4053af2.png)
就这里的tab页里嵌套自定义组件

**Reply by janryWang** [reply]
不要Form套Form，直接内部用RecusionField就行了

**Reply by PlutoCA** [reply]
ok了， 感谢~
